src/AWSSQS.jl

```diff
@@ -16,15 +16,15 @@ export sqs_list_queues, sqs_get_queue, sqs_create_queue, sqs_delete_queue,
        sqs_set_policy, sqs_name, sqs_arn,
        sqs_send_message, sqs_send_message_batch, sqs_receive_message,
        sqs_delete_message, sqs_flush, sqs_get_queue_attributes, sqs_count,
-       sqs_busy_count
+       sqs_busy_count, sqs_messages
```

Comment:
I'm hesitant to export the iterator because it has no test cases and needs a bit more work to be user friendlily. At present, it doesn't behave nicely if there are no more messages.
Note: https://github.com/samoconnor/AWSSQS.jl/commit/beb67ce12ad3df8f9e505002a33ace762f31b8b3

If you're using it and would like to write some test cases that would be great!